appinventor/blocklyeditor/src/versioning.js

```diff
@@ -1027,6 +1027,7 @@ Blockly.Versioning.getOptionListValueMap = function(workspace, key) {
   var optionList = db.getOptionList(key);
   for (var i = 0, option; option = optionList.options[i]; i++) {
     map[option.value] = option.name;
+    map[option.value.toLowerCase()] = option.name;
```

Comment:
Some of the old values would accept either lowercase values or uppercase values. Eg for motor ports you could have 'A' or 'a'. So upgrading should support this as well.

Rather than making a separate function for enums like NxtMotorPort (that support both cases) I just added it to this function, because I figured there was no harm in being forgiving.

If you like I can create a separate function that allows for both cases.

Reply:
I think this is fine because I don't expect there are other places in the code where you'd have different semantics for items that differ only by case.